pilot/pkg/features/pilot.go

```diff
@@ -157,6 +157,14 @@ var (
 		false,
 		"If enabled, controller that untaints nodes with cni pods ready will run. This should be enabled if you disabled ambient init containers.").Get()
 
+	ConnectOriginateIdleTimeout = env.Register(
+		"PILOT_CONNECT_ORIGINATE_IDLE_TIMEOUT",
+		0*time.Second,
+		"Idle timeout for HBONE connections to ztunnel. "+
+			"Default is 0 (disabled), which uses Envoy's default 1-hour timeout. "+
```

Comment:
This is kinda a nit, but what do people think of having the default be 3600, that way we avoid that `if ConnectOriginateIdleTimeout > 0) logic, and people can set the ConnectionOriginateIdleTimeout to 0.